src/pages/NewChatPage.js

```diff
@@ -250,6 +250,8 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, translate, i
                             shouldPreventDefaultFocusOnSelectRow={!Browser.isMobile()}
                             shouldShowOptions={isOptionsDataReady}
                             shouldShowConfirmButton
+                            shouldShowCTA
```

Comment:
Just a suggestion to make this more specific. 
```suggestion
                            shouldShowReferralCTA
```